product-page.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://water-style-ui.netlify.app/main.css">
+    <link rel = "stylesheet" href = "styles.css">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Products</title>
+</head>
+<body>
+
+    <nav class = "navigation flex gap-xl align-center full-width">
+        <div class="nav-brand">Water-Commerce</div>
+        <div class = "input-wrapper flex grow-1 flex-center" >
+           <input class = "input-field" placeholder="Please Enter Text">
+       </div>
+       <div class = "flex gap-xl flex-center grow-1" >
+           <button class="btn btn-secondary">Login</button>
+           <i class="material-icons" >favorite_border</i>
+           <i class="material-icons" >shopping_cart</i>
+       </div>
+   </nav>
+
+   <div class="wrapper flex no-wrap">
+       <aside class = "side-bar">
+           <div class = "flex space-between padding-m">
+               <strong>Filter</strong>
+               <p>Clear</p>
+           </div>
+
+           <div class = "padding-m m2-top">
+               <div class = "text-s bold">Price</div>
+               <div>
+                    <input type="range" min="1" max="100" value="50" class = "slider m2-top full-width">
+               </div> 
+           </div>
+
+           <div class = "padding-m m2-top">
+                <div class = "text-s bold">Category</div>
```

Comment:
text-s is a class not easy to understand, you can use a better name here

Reply:
So for text I have made classes like text-xs,text-s,text-m and so on

Reply:
Thanks for the suggestion though